### PR TITLE
Support for `inet` data type.

### PR DESCRIPTION
**Pull request by rikvdkleij**
Using version 0.10 of `postgresql-binary` and `network-ip` package.

### CODE REVIEW

File: hasql.cabal [hunk 1]
```diff
@@ -74,7 +74,7 @@ library
     -- parsing:
     attoparsec >= 0.10 && < 0.14,
     -- database:
-    postgresql-binary >= 0.9.1.1 && < 0.10,
+    postgresql-binary >= 0.10 && < 1,
```
Comment (nikita-volkov) [inline hasql.cabal]: There's no guarantee that the next major-version bump won't be API-breaking even before `1`. So it needs to be restricted to `0.11`.

File: hasql.cabal [hunk 2]
```diff
@@ -89,6 +89,7 @@ library
     text >= 1 && < 2,
     bytestring >= 0.10 && < 0.11,
     hashable >= 1.2 && < 1.3,
+    network-ip >= 0.2 && < 1,
```
Comment (nikita-volkov) [inline hasql.cabal]: Same as with "postgresql-binary". The upper bound needs to be `0.3`.

File: hasql.cabal (hunk 1 above)
Comment (rikvdkleij) [inline hasql.cabal]: Okay, my mistake, I assumed the next major version would be 1.0. I will change it.

File: hasql.cabal (hunk 2 above)
Comment (rikvdkleij) [inline hasql.cabal]: Okay, will change it.

Comment (rikvdkleij) [inline hasql.cabal]: I used 0.3 version locally with GHC 8.0.2. Version 0.2 is necessary for older GHC versions. So l will change upper bound to 0.4

**Comment by nikita-volkov** [comment]
Good job! It's released.

**Comment by rikvdkleij** [comment]
Thanks!

**Pull request merged**